aspnetcore/grpc/netstandard.md

```diff
@@ -69,6 +69,8 @@ For more information, see [Configure gRPC-Web with the .NET gRPC client](xref:gr
 Requirements and restrictions to using `WinHttpHandler`:
 
 * Windows 11 or later, Windows Server 2022 or later.
+  * gRPC client is fully supported on Windows 11 or later.
+  * gRPC client is partially supported on Windows Server 2022. Unary and server streaming methods are supported. Client and bidirectional streaming methods are **_not_** supported.
```

Comment:
qq: is the mode of failure for "not supported" functionality documented somewhere else than the code itself?

Basically, 
1. What happens when you try any gRPC call on platform that doesn't support WinHttpHandler at all.
2. what happens when you attempt client streaming or bi streaming call on a system where that's not supported.
3. what happens when you try non-TLS with WinHttpHandler

I don't think this needs to be documented here specifically, but may well be worth mentioning somewhere else.

Reply:
> What happens when you try any gRPC call on platform that doesn't support WinHttpHandler at all.

I think WinHttpHandler throws a platform not supported exception that Windows is required.

> what happens when you attempt client streaming or bi streaming call on a system where that's not supported.
> what happens when you try non-TLS with WinHttpHandler

I recall that in both these cases WinHttpHandler silently falls back to sending an HTTP/1.1 request which then fails.